setup.py

```diff
@@ -29,6 +29,8 @@ def read(*paths):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
FYI --

https://wiki.debian.org/Python
Debian 9 uses Python 3.5 and Debian 10 uses Python 3.7